corehq/form_processor/utils/sql.py

```diff
@@ -61,6 +61,10 @@ def form_adapter(form):
         form.deleted_on,
         form.deletion_id,
         form.server_modified_on,
+        form.time_start,
+        form.time_end,
+        form.commcare_version,
+        form.build_version
```

Comment:
@snopoke I couldn't find which if any SQL functions that need an update, are there any?

Reply:
👍